app/javascript/styles/_uploadWizard.scss

```diff
@@ -147,3 +146,7 @@
 .prev-next-buttons {
   margin-left: auto;
 }
+
+.margin-left {
+  margin-left: 15px;
+}
```

Comment:
```suggestion
}

```

Reply:
Ending files with newlines is an SCP code convention.  I believe there's a VS Code setting for it.